app/models/miq_provision.rb

```diff
@@ -31,7 +31,7 @@ class MiqProvision < MiqProvisionTask
 
   CLONE_SYNCHRONOUS     = false
   CLONE_TIME_LIMIT      = 4.hours
-  SUPPORTED_EMS_CLASSES = %w(ManageIQ::Providers::Vmware::InfraManager ManageIQ::Providers::Redhat::InfraManager ManageIQ::Providers::Amazon::CloudManager ManageIQ::Providers::Openstack::CloudManager ManageIQ::Providers::Microsoft::InfraManager)
+  SUPPORTED_EMS_CLASSES = %w(ManageIQ::Providers::Vmware::InfraManager ManageIQ::Providers::Redhat::InfraManager ManageIQ::Providers::Amazon::CloudManager ManageIQ::Providers::Openstack::CloudManager ManageIQ::Providers::Microsoft::InfraManager ManageIQ::Providers::Azure::CloudManager)
```

Comment:
@matthewd in a future PR, would be good to have this list be created via registration or composition, instead of hard-coding class names.